src/Cameras/video_streaming/launch/video_streaming.launch.py

```diff
@@ -34,6 +34,16 @@ def generate_launch_description():
         parameters=[],
     )
 
+    srt_node = ComposableNode(
+        package="video_streaming",
+        plugin="video_streaming::SrtNode",
+        name="srt_node",
+        namespace="",
+        parameters=[
+            {"srt_uri": "srt://127.0.0.1:7001", "latency": 100, "iframe_interval": 30}
```

Comment:
I would say it make more sense for the rover (aka "Sender") to be an SRT listener. That way we can choose to have multiple callers and it doesn't matter what IP the address of the client is. I believe all you have to do is switch the uri to not include the ip address and just have the port.
```suggestion
            {"srt_uri": "srt://:7001", "latency": 100, "iframe_interval": 30}
```